Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -388,12 +388,12 @@ internal void ClientChangeScene(string newSceneName, bool forceReload)
             }
 
             // vis2k: pause message handling while loading scene. otherwise we will process messages and then lose all
-            // the sate as soon as the load is finishing, causing all kinds of bugs because of missing state.
+            // the state as soon as the load is finishing, causing all kinds of bugs because of missing state.
             // (client may be null after StopClient etc.)
             if (client != null)
             {
                 if (LogFilter.Debug) { Debug.Log("ClientChangeScene: pausing handlers while scene is loading to avoid data loss after scene was loaded."); }
-                NetworkClient.pauseMessageHandling = true;
+                NetworkManager.singleton.transport.Pause();
```

Comment:
this is nice

Reply:
yes,  I think this makes it much clearer what is being paused.